delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/syntax/package.scala

```diff
@@ -14,3 +14,4 @@ package object syntax
     with IriUriSyntax
     with IterableSyntax
     with KamonSyntax
+    with IOResourceFSyntax
```

Comment:
This `IOResourceFSyntax ` has to be added as well to `ch.epfl.bluebrain.nexus.delta.syntax`